guide/interactions/buttons.md

```diff
@@ -113,10 +113,10 @@ client.on('interactionCreate', async interaction => {
 If you want to prevent a button from being used, but not remove it from the message, you can disable it with the `setDisabled()` method:
 
 ```js {5}
-const button = new MessageButton()
+const button = new Builder()
```

Comment:
new Button() should be new ButtonBuilder()

Reply:
fixed :s